lib/node_modules/@stdlib/math/base/complex/divide/README.md

```diff
@@ -0,0 +1,80 @@
+# divide
+
+> Divide two complex numbers.
+
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var cdiv = require( '@stdlib/math/base/complex/divide' );
+```
+
+#### cdiv( re1, im1, re2, im2 )
+
+Divides two `complex` numbers where each `complex` number is comprised of a __real__ component `re` and an __imaginary__ component `im`.
+
+``` javascript
+var v = cdiv( 5.0, 3.0, -2.0, 1.0 );
+// returns [ -13.0, -1.0 ]
+```
+
+</section>
+
+<!-- /.usage -->
+
+
+<section class="examples">
+
+## Examples
+
+``` javascript
+var Complex128 = require( '@stdlib/types/complex/float64' );
+var randu = require( '@stdlib/math/base/random/randu' );
+var round = require( '@stdlib/math/base/special/round' );
+var real = require( '@stdlib/types/complex/real' );
+var imag = require( '@stdlib/types/complex/imag' );
+var cdiv = require( '@stdlib/math/base/complex/add' );
+
+var re;
+var im;
+var z1;
+var z2;
+var z3;
+var o;
+var i;
+
+for ( i = 0; i < 100; i++ ) {
```

Comment:
Is this example useful? Could I just use the simple:

```javascript
var cdiv = require( '@stdlib/math/base/complex/divide' );

cdiv( ... );
// => [...]
```

style example instead?